test/functional/wallet_basic.py

```diff
@@ -496,6 +496,11 @@ def run_test(self):
         self.nodes[0].setlabel(change, 'foobar')
         assert_equal(self.nodes[0].getaddressinfo(change)['ischange'], False)
 
+        # Test "decoded" field value in gettransaction response
```

Comment:
this is a weird nit, but just curious why there's a "Non-breaking space" character between the `#` and `Test` (utf-8 0xC2A0)

```
git cat-file -p d5ab374c48a7f2708acc5e122044a3a9cfc1ac6a | grep '#.Test "decoded"' | xxd | head -n1
```

Reply:
I think it's caused by a vim module (deoplete-jedi on this one I think)